eq_cir_converter_service/routers/schema_router.py

```diff
@@ -36,6 +36,10 @@ async def post_schema(
     Request body:
     - schema: The schema to convert.
 
+    Steps:
```

Comment:
We had a brief discussion about this, and @farres1 made an improvement by replacing multiline comment tags with single-line comments.